src/test/kotlin/fi/metatavu/megasense/dataportal/api/test/functional/builder/auth/TestBuilderAuthentication.kt

```diff
@@ -128,4 +130,28 @@ class TestBuilderAuthentication(testBuilder: AbstractTestBuilder<ApiClient>, acc
 
         return medicalConditions!!
     }
+    /**
+     * Returns a test builder resource for pollutants API
+     *
+     * @return a test builder resource for pollutants API
+     */
+    fun pollutants(): PollutantTestBuilderResource {
+        if (pollutants == null) {
+            pollutants = PollutantTestBuilderResource(testBuilder, accessTokenProvider, createClient())
+        }
+
+        return pollutants!!
+    }
+    /**
```

Comment:
```suggestion
    }
    
    /**
```
Empty line is missing